Community/Tdarr_Plugin_2sus_Burn_in_Subtitles_Handbrake.js

```diff
@@ -0,0 +1,172 @@
+/* eslint-disable */
```

Comment:
Plugins need to be linted and have tests written.

Reply:
I enabled linting and resolved all the issues it found